index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="hamburger">
+        <div class='head'> 
+            <div class="name">Yohannes Zemed </div>
+           <img src="icons/menu.png" alt='menu'/>
+        </div>
+    </header>
+    <section class="heading">
+      <div class="main-text">
+        <div class ='text'>
+         <h1>Hey There.</h1>
+         <h1>I'm Yohannes</h1>
+        </div>
+        <p class ='text1'>I'am a Software Developer</p>
+        <p class ='text2'>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+       </div>
+       <ul class='contacts'>
+          <li> <a href='https://github.com/Yoz21'><img src="icons/github.svg" alt="github addres" /></a></li>
+           <li><a href='https://angel.co'><img src="icons/angelist.png" alt='angelist addres'/><a</li>
+           <li><a href='https://medium.com'><img src="icons/medium.png"  alt='medium addres'/></a></li>
+           <li><a hef='https://www.linkedin.com/feed/'><img src="icons/Linkedin icon.svg" alt="linkedin addres"></a></li>
+           <li><a href='https://twitter.com/Yoz44297468'><img src="icons/twitter.png" alt="twitter address"></a></li>
+
+       </ul>
```

Comment:
- [ ] OPTIONAL: I suggest you kindly correct the icons so they are consistent in their alignment, and size:

![image](https://user-images.githubusercontent.com/57421823/148509873-ebc19e41-944d-4125-9ee9-e523c735c282.png)

This can be achieved with the use of `height` and `width` properties (eg. all of them the same width and height), and the use of flexbox for alignment.

That way, your icons will look more consistent, and professional hugely improving the overall look of your portfolio! 💪